addons/kommander/1.1.2/kommander-1.yaml

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
It's not taken full effect yet but the plan in the (near) future is that we'll only have one revision file per release as kubeaddons supports that (this repo used to do this before). So we can have a plain ol `kommander.yaml` file in a version folder. We'll still need to bump the `addon-revision` on L9 for but we'd save a step (and ease of review) by getting rid of that first 1) copy/prep revision file step

see: https://github.com/mesosphere/ksphere-platform/pull/192